src/docfx/localization/LocalizationUtility.cs

```diff
@@ -2,18 +2,24 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Collections.Generic;
 using System.Diagnostics;
 using System.Globalization;
 using System.IO;
+using System.Linq;
 using System.Text.RegularExpressions;
 
 namespace Microsoft.Docs.Build
 {
     internal static class LocalizationUtility
     {
+        private static readonly HashSet<string> s_locales = new HashSet<string>(CultureInfo.GetCultures(CultureTypes.AllCultures).Except(CultureInfo.GetCultures(CultureTypes.NeutralCultures)).Select(c => c.Name), StringComparer.OrdinalIgnoreCase);
```

Comment:
is `Contains('-')` a better condition to ensure we don't include strings without `-` into the locale list?

Reply:
I am wondering how is this list compared to the actual list Docs supports...

Reply:
I think it's the same from the summary:
```
  Cultures that are associated with a language but are not specific to a country/region.
        //     The names of .NET Framework cultures consist of the lowercase two-letter code
        //     derived from ISO 639-1. For example: &quot;en&quot; (English) is a neutral culture.
        NeutralCultures = 1,

Reply:
> I am wondering how is this list compared to the actual list Docs supports...

I added docs support locales to the test

Reply:
I would like to suggest we start with the docs locale list

Reply:
don't want to add docs' limitation to docfx.